### PR TITLE
Add timetable for the COVID-19 period

### PR DESCRIPTION
**Pull request by YusunPark**
- The timetable for the semester has been changed because of the Corona virus.
  - Week
    - 08:00~19:00 =>10-minute interval
    - 19:00~22:00 =>30-minute interval
  - Weekend
    - 08:00~22:00 =>60-minute interval

### CODE REVIEW

**Comment by CXZ7720** [comment]
The directory name must not be changed. remove COVID at directory name.

**Review by CXZ7720 — CHANGES_REQUESTED** [review]
do not change the source directory name

**Comment by YusunPark** [comment]
I named it 'semester_covid19' because it was a temporary timetable.
I'll change it.

File: timetable/semester/weekend/Shuttlecock_I_weekend.json [hunk 1]
```diff
@@ -1,58 +1,18 @@
 {
   "shuttlecock_i_weekend": [
```
Comment (Taewan-P) [inline timetable/semester/weekend/Shuttlecock_I_weekend.json:2]: This schedule is for Shuttlecock inbound. All the types should be "R", meaning Residence.

File: timetable/semester/weekend/Residence_weekend.json [hunk 2]
```diff
@@ -1,53 +1,12 @@
 {
   "residence_weekend": [
-    { "time": "07:45", "type": "C" },
```
Comment (Taewan-P) [inline timetable/semester/weekend/Residence_weekend.json:3]: .

File: timetable/semester/week/YesulIn_week.json [hunk 3]
```diff
@@ -1,48 +1,33 @@
...
-    { "time": "21:25", "type": "" },
     { "time": "21:35", "type": "" },
-    { "time": "21:45", "type": "" },
-    { "time": "22:00", "type": "" },
-    { "time": "22:15", "type": "" },
-    { "time": "22:30", "type": "" },
-    { "time": "22:45", "type": "" },
-    { "time": "23:00", "type": "" },
-    { "time": "23:15", "type": "" }
+    { "time": "22:05", "type": "" }
```
Comment (Taewan-P) [inline timetable/semester/week/YesulIn_week.json:31]: This one too. We need to figure out whether 2205 is a valid bus schedule.

**Review by Taewan-P — CHANGES_REQUESTED** [review]
As requested.

We are really not sure about the 2205 bus & weekend bus start time for now.

**Review by Taewan-P — CHANGES_REQUESTED** [review]
* Found out that weekend shuttle starts at 8.
* Buses at shuttlecock_i that arrives before 10 should be marked as type "NA"

File: timetable/semester/week/YesulIn_week.json (hunk 3 above)
Comment (Taewan-P) [reply timetable/semester/week/YesulIn_week.json:31]: Found out this 2205 is valid.

**Review by Taewan-P — APPROVED** [review]
LGTM.

File: timetable/semester/weekend/YesulIn_weekend.json [hunk 4]
```diff
@@ -1,6 +1,5 @@
 {
   "yesulin_weekend": [
-    { "time": "08:05", "type": "" },
```
[the remark this answers is not in the document]
Comment (Taewan-P) [reply timetable/semester/weekend/YesulIn_weekend.json]: Starts at 8

**Review by Taewan-P — APPROVED** [review]

**Review by CXZ7720 — APPROVED** [review]

**Pull request merged**